docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+```
+
+The `configtxgen` tool used the `TwoOrgsOrdererGenesis` channel profile from `configtx.yaml` to write the genesis block and store it in the `system-genesis-block` folder. You can see the `TwoOrgsOrdererGenesis` profile below:
+```yaml
+TwoOrgsOrdererGenesis:
+    <<: *ChannelDefaults
+    Orderer:
+        <<: *OrdererDefaults
+        Organizations:
+            - *OrdererOrg
+        Capabilities:
+            <<: *OrdererCapabilities
+    Consortiums:
+        SampleConsortium:
+            Organizations:
+                - *Org1
+                - *Org2
+```
+
+The `Orderer:` section of the profile creates the single node Raft ordering service used by the test network, with the `OrdererOrg` as the ordering service administrator. The `Consortiums` section of the profile creates a consortium of peer organizations named `SampleConsortium:`. Both peer organizations, Org1 and Org2, are members of the consortium. As a result, we can include both organizations in new channels created by the test network. If we wanted to add another organization as a channel member, we would first need to create the channel with Org1 and Org2, and then add the other organization by [updating the channel configuration](../channel_update_tutorial.html).
+
+## Creating an application channel
+
+Now that we have deployed the nodes of the network and created the orderer system channel using the `network.sh` script, we can start the process of creating a new channel for our peer organizations. We have already set the environment variables that are required to use the `configtxgen` tool. Run the following command to create a channel creation transaction for `channel1`:
+```
+configtxgen -profile TwoOrgsChannel -outputCreateChannelTx ./channel-artifacts/channel1.tx -channelID channel1
+```
+
+The `-channelID` will be the name of the future channel. Channel names must be all lower case, less than 250 characters long and match the regular expression ``[a-z][a-z0-9.-]*``. The command uses the uses the `-profile` flag to reference the `TwoOrgsChannel:` profile from `configtx.yaml` that is used by the test network to create application channels:
+```yaml
+TwoOrgsChannel:
+    Consortium: SampleConsortium
+    <<: *ChannelDefaults
+    Application:
+        <<: *ApplicationDefaults
+        Organizations:
+            - *Org1
+            - *Org2
+        Capabilities:
+            <<: *ApplicationCapabilities
+```
+
+The profile references the name of the `SampleConsortium` from the system channel, and includes both peer organizations from the consortium as channel members. Because the system channel is used as a template to create the application channel, the ordering service defined in the system channel become the default [consenter set](../glossary.html#consenter-set) of the new channel. Ordering nodes and ordering organizations can be added or removed from the consenter set using channel updates.
```

Comment:
> Because the system channel is used as a template to create the application channel, the ordering service defined in the system channel become the default [consenter set](../glossary.html#consenter-set) of the new channel. Ordering nodes and ordering organizations can be added or removed from the consenter set using channel updates.

This is a bit confusing to me.  Why do we emphasize that the set of consenters is inherited from the orderer system channel, but not the otherproperties? For instance the batch size and timeout, as well as the orderer orgs and their crypto material and policies (as well as the crypto material and policies of the application orgs from the consortium definition.

FWIW, it is possible to modify these through the channel creation tx (via that whole `-baseProfile` thing), but it's pretty cludgy, we might want to avoid documenting it and instead steer people towards the channel participation API once it's out.

Reply:
Some of the more advanced parameters, like the batch size and timeout, are best left to the second tutorial.

However, I did add language talking about the orderer administrators in addition to the ordering nodes: 
 Because the system channel is used as a template to create the application channel, the ordering nodes defined in the system channel become the default [consenter set](../glossary.html#consenter-set) of the new channel, while the administrators of the ordering service become the orderer administrators of the channel.